### ISSUE TITLE
Skype dumped me

### ISSUE THREAD

**Issue by eric-brechemier** [eric-brechemier/how-i-replaced-skype-with-twilio#1]
One week before Christmas, on 18 December 2017, my subscription to Skype premium service "France Unlimited mins to landlines" was cancelled with no action from my part and no prior warning. I had been subscribed without interruption for more than 9 years, since July 24th 2008.

"We are sorry to hear that you will no longer be using our subscription service. We hope you enjoyed using it.", they said in the email that I received on that day:

---

![2017-12-18-we-are-sorry-to-hear](https://user-images.githubusercontent.com/151276/54434285-2d902580-472e-11e9-8520-b582a86cd693.png)

---

This sounded like I had cancelled the subscription myself, but I clearly didn't, and I was not aware of any payment error either. This was obviously a mistake. I connected to Skype dashboard, where I found no further hint, but confirmed that the subscription was gone. To make things worse, the attached phone number, which I used all over the place including in my resume for the better part of the past ten years, would get into oblivion in less than a month:

---

![2018-01-04-your-skype-number-will-expire](https://user-images.githubusercontent.com/151276/54435137-1ce0af00-4730-11e9-8146-0dafc59d2422.png)

---

I contacted Skype support. It was all normal, they said. This subscription had been discontinued and all affected users have been notified. Well, obviously not all, I answered, or I would have known. Well, now you are notified, they said. This subscription is no longer available. How is it possible? What do you propose? I asked. I am going to lose my phone number. I tried to renew the subscription or even just the phone number, but nothing works. And I can't get a new subscription for this number because it is marked as unavailable.

This is all normal, they said. You have to wait until the subscription expires. Then the phone number will become available again, and you can get a new subscription.

I don't understand, I said. How much is it going to cost to get something equivalent to what I had before?

Let me see, they said. Your Country subscription, "France Unlimited minutes to landlines", used to cost 3.54 EUR/month. If you still wish to have unlimited minutes to call landlines in France, there is a new subscription which costs 3.60 EUR/month, but does not include a Skype Number like your previous subscription did. You will have to purchase a Skype Number separately. Ok, I said.

How much is it going to cost then? I asked. I can't find the price for a Skype Number on your website. I need to select a new number and agree to conditions of use before I can get the price. But I wish to keep my existing number. Is that possible? Sure, they said. You can use any Skype Number with any subscription. Well, how much is it going to cost, then? I asked. I cannot tell you, they said. The monthly fee depends on the type of number and the location. Then you get a different discount if you subscribe for 3 months, 6 months or 12 months. That is annoying, I said. How come that I cannot see the price for a Skype Number before I accept the terms of the subscription? You just have to wait until your old subscription expires, they said. You can then renew your Skype Number and you will see the price. Make sure to subscribe for 12 months and you will get a 50% discount.

Speaking of discounts, I said, and given how my current subscription was cancelled unilaterally by Skype, I would really appreciate to get a discount on the renewal of my Skype Number over the next year, to cover the extra cost induced by the unanticipated change of subscription. That is impossible, they said. You are not leaving me much of a choice, I said. May I speak with your manager? No, that is impossible as well, they answered.

**Comment by eric-brechemier** [comment]
My subscription expired on January the 3rd, 2018.

> Skype is committed to providing you with low cost calling rates. We regularly review our subscriptions to make sure we're giving our customers the best value. Occasionally subscriptions may become unavailable through one of our partners or we may discontinue some subscriptions.
>
> If your subscription is discontinued or becomes unavailable, we will notify you in your Skype account, or by other reasonable means, that your subscription will not be renewed. It will remain active until its expiry date, but it won't automatically renew and you won't be able to re-purchase that subscription. If you want to continue making low-cost calls with a calling plan, you will need to choose a new one.
> — https://support.skype.com/en/faq/FA34707/why-is-my-subscription-no-longer-available

**Comment by eric-brechemier** [comment]
My Skype Number expired on January 11th, 2018.

> After it expires, Skype will reserve your Skype Number for you on your account for 90 days. This means no one else but you can purchase this particular number during this period. You can renew your Skype Number subscription from your account.
> — https://support.skype.com/en/faq/FA12164/how-do-i-cancel-my-skype-number

**Comment by eric-brechemier** [comment]
I decided to stop using Skype to make phone calls to regular phones, and to start looking for an alternative provider for a VOIP phone number that I can publish online and share with all the pesky services that now require one during signup.

I did not pick any new Skype Subscription. I decided, however, to renew the Skype Number for an extra year, to give me the time to find an alternative.

#### Reference
* [How is a subscription different to a Skype Number?](https://support.skype.com/en/faq/FA12410/how-is-a-subscription-different-to-a-skype-number?q=How+is+a+subscription+different+from+a+Skype+Number)

**Comment by eric-brechemier** [comment]
Apart from the obvious, namely that Skype ended our 10-year relationship with no remorse and not a word of explanation, my decision was compounded by the fact that over the years, the Skype to Phone service saw little progress in terms of features, but greatly deteriorated in terms of user experience, especially in the shiny new version of the desktop app released recently before.

**Comment by eric-brechemier** [comment]
## Missing Skype to Phone Features

### Phone

I used to provide my Skype number during signup when a phone number is required because I don't want to get bothered by robocalls or other unsolicited calls. But recently, these services started to require explicitly a mobile phone number, starting with +336 or +337 in France, while Skype only offers so-called "non-geographical (VOIP)" numbers in France, starting with +339. As such, a Skype Number is often rejected these days in signup forms that require a mobile phone number.

### SMS

* in France and other countries, Skype does not advertize the number that I configured in Caller ID. SMS messages sent with Skype appear from a generic number instead (e.g. `37208`)
* as a result, Skype can send SMS messages, but you cannot receive any reply, not even on a different phone

**Comment by eric-brechemier** [comment]
## Degraded User Experience in Skype Desktop (2017)

In 2017, Skype turned 14, which is 88 in dog years. Fittingly, they offered a major redesign to their desktop app, which, by copying the looks of mobile apps in vogue at the time, in a failed attempt to appeal to a younger audience, made it look like an 88-year-old who starts to dress like a teenager:

---

<img width="220" alt="2017-12-04-Skype-horror-1" src="https://user-images.githubusercontent.com/151276/54455517-529b8d00-475c-11e9-935a-6a8de398e410.png"> <img width="220" alt="2017-12-04-Skype-horror-2" src="https://user-images.githubusercontent.com/151276/54458426-3865ad00-4764-11e9-811b-806c6e1bc109.png"> <img width="220" alt="2017-12-04-Skype-horror-3" src="https://user-images.githubusercontent.com/151276/54458242-aa89c200-4763-11e9-84bb-1ee4eec4aeef.png">

---

Icons without text. First name and initials guessed incorrectly from full (display) names. A total lack of contrast even using the dark theme. No hierarchy of information and barely any information displayed on the screen.

It was nearly unusable.

**Comment by eric-brechemier** [comment]
On January 22, 2018, I renewed my Skype Number for 12 months, to benefit from a 50% discount on the same subscription paid monthly. In spite of that discount, the 12 month subscription cost 41.04€ for a Skype Number alone, while my previous subscription included a Skype Number and unlimited minutes to landlines for 42.48€ per year (3.54€/month).

In order to get the same level of service, I would have had to go from:

| Subscription | France Unlimited mins to landlines |
| :--- | --- |
| Status | *Cancelled at End of 2017* |
| Skype Number | ✓ |
| Calls to landlines | ✓ |
| Payment | Monthly |
| Yearly Cost | 42.48€ |

to

| Subscription | Skype Number | France Unlimited mins to landlines |
| :--- | --- | --- |
| Status | *Available* | *Available* |
| Skype Number | ✓ | 𐄂 |
| Calls to landlines | 𐄂 | ✓ |
| Payment | Yearly | Monthly |
| Yearly Cost | 41.04€ | 43.20€ |

In short, I could now have the same service as before for double the price.

**Comment by eric-brechemier** [comment]
When my Skype Number subscription expired a year later, on January 22, 2019, I did not renew it.

**Comment by eric-brechemier** [comment]
You may leave your comments below.

**Comment by lucanos** [comment]
I had Skype pull a similar scam on me. I had the Unlimited World Subscription, which included 3 phone numbers. And I had also made the mistake of loading my Skype account up with credit (from gift cards which I would buy when they were on special). So I ended up in a position where my subscription was cancelled without my consent, with enough credit on my account to cover 3 years of what I had been paying, and in a situation where buying the new products for the same level of service would have cost me 3 times what I had been paying.

Needless to say, I am just using that remiaining credit for the odd call or SMS, especially when overseas, and will also be looking to migrate all of my online telephony needs away from Skype. It's also pretty obvious that, since Microsoft bought Skype, they were only really looking to integrate the tech into what was Lync and is now Teams - they have no real interest in maintaining it as a standalone product.
